src/modules/previewpane/PreviewPaneUnitTests/MarkDownPreviewHandlerHTMLParsingExtensionTest.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Markdig;
+using MarkDownPreviewHandler;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace PreviewPaneUnitTests
+{
+    [TestClass]
+    public class MarkDownPreviewHandlerHTMLParsingExtensionTest
+    {       
+        public MarkdownPipeline TestBase(IMarkdownExtension extension)

```

Comment:
Should this be private?  Also another 'nit' comment, but the method name doesn't really seem to describe what it does.  Maybe something like BuidPipelineWithExtention or just BuildPipeline as the 'with extension' is implied by the signature.